### PR TITLE
Add travis and reorder ut folders

### PR DESCRIPTION
**Pull request by shargon**

### CODE REVIEW

**Comment by shargon** [comment]
More about general guideline, split source from tests

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+mono: none
+dotnet: 2.1.300
+
+before_install:
+  - cd tests/neo-vm.Tests
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ulimit -n 2048; fi
+
+script:
+ - dotnet restore
+ - dotnet test
```
Comment (igormcoelho) [inline .travis.yml]: That's fine for me, I never done this configuration before. It will be very helpful!

Comment (shargon) [reply .travis.yml]: What configuration do you use?

Comment (igormcoelho) [reply .travis.yml]: I mean, never configured travis before.

**Pull request merged**